WireDomain/Sources/WireDomain/Helpers/InitiateResetMLSConversationUseCase.swift

```diff
@@ -47,14 +57,16 @@ public class InitiateResetMLSConversationUseCase: InitiateResetMLSConversationUs
 
         do {
 
-            guard let conversation = await conversationLocalStore.fetchMLSConversation(
-                groupID: groupID
-            ) else {
+            guard let conversation = await conversationLocalStore.fetchMLSConversation(groupID: groupID),
+                  let qualifiedID = await conversationLocalStore.qualifiedID(for: conversation)
+            else {
                 WireLogger.mls.error("Initiate reset broken MLS conversation failed: no conversation found")
                 return
             }
-            let qualifiedID = await conversationLocalStore.qualifiedID(for: conversation)
-            attributes = [.conversationId: qualifiedID?.safeForLoggingDescription ?? "<nil>"]
+
+            attributes = [.conversationId: qualifiedID.safeForLoggingDescription]
+
+            userDefaultsRepository.setInitiatedReset(conversationID: qualifiedID)
```

Comment:
issue: we never reset the flag here.
```suggestion
            userDefaultsRepository.setInitiatedReset(conversationID: qualifiedID)
            defer {
            userDefaultsRepository.removetInitiatedReset(conversationID: qualifiedID)
            }
```

I guess it's kind of the same as removing the flag in the processEvent... what do you think about this instead?

Reply:
It's removed when trying to proccess it. As I can understand, event can come after when Initiate step completed, e.g. because of slow network. If we remove it here event in this case will be handled on same device from which it was triggered